document/contract.go

```diff
@@ -20,14 +20,14 @@ func ContractFx(w http.ResponseWriter, r *http.Request) (string, interface{}, er
 	defer r.Body.Close()
 	err := json.Unmarshal([]byte(req), &data)
 	lib.CheckError(err)
-	respObj := <-ContractObj(origin, data)
+	respObj := <-ContractObj(origin, data, nil)
```

Comment:
ok if this function is not triggered externally, otherwise we should extract the node from the authToken